src/routes/login.js

```diff
@@ -0,0 +1,59 @@
+import express from 'express';
```

Comment:
@ Victor remember to rename this file to "auth" referring to what Temitayo suggested.Thx